data/docs/collection-agents/k8s/otel-operator/overview.mdx

```diff
@@ -56,12 +56,33 @@ The operator can inject and configure [OpenTelemetry auto-instrumentation](https
 
 To manage automatic instrumentation, the Operator must be configured to identify which pods to instrument and select the appropriate automatic instrumentation for those pods. This configuration is done through the [Instrumentation CRD](https://github.com/open-telemetry/opentelemetry-operator/blob/main/docs/api/instrumentations.md) and adding [Annotations](http://localhost:3000/docs/collection-agents/k8s/otel-operator/install/#annontations-for-auto-instrumentation-injection) to the workloads
 
-<Admonition type="tip">
-**`k8s-infra`** is a pre-configured Helm chart from SigNoz, tailored for Kubernetes infrastructure observability. It provides a straightforward, out-of-the-box solution: simply install the chart in your cluster, and it will automatically gather metrics, logs, traces, and events from your entire Kubernetes environment and no manual configuration needed.
+## K8s-Infra vs OpenTelemetry Operator
+
+When deciding between SigNoz’s **k8s-infra** Helm chart and the **OpenTelemetry Operator**, the choice often comes down to setup simplicity versus flexibility.  
+
+### Why and When k8s-infra
+
+- **Fast Setup**: Install a single Helm chart, and it auto-collects metrics, logs, traces, and events.  
+- **SigNoz-Ready**: Pre-configured for SigNoz (dashboards, alerts, integrations work out of the box).  
+- **Minimal Overhead**: No need to write collector configs or manage multiple CRDs.  
+- **Maintenance**: Updates and patches delivered via Helm chart releases.  
+
+Use **k8s-infra** when you want a quick, production-ready way to start sending Kubernetes telemetry to SigNoz without learning OTel collector internals. For more information, refer to the [K8s-infra Collection Agent](https://signoz.io/docs/collection-agents/k8s/k8s-infra/overview/).
+
+### Why and When OpenTelemetry Operator
```

Comment:
As we discussed, we only need to mention 1 core reason of Auto instrumentation.
Also Custom collector configs are available even with k8s-infra chart, right?